db/beta_seed/curriculum.rb

```diff
@@ -131,7 +131,7 @@ def create_curriculum(number_of_semesters, semester_type)
       unit_id:     unit.id,
       program_ids: programs.pluck(:id)
     )
-    curriculum.build_semesters(number_of_semesters: number_of_semesters, type: semester_type)
+    curriculum.build_semesters
```

Comment:
Bu dosyada bu parametreler birkaç yerde daha kullanılıyor. O yüzden onları silersek daha temiz kod olur.

Reply:
evet travis'ten gelen hata sonrasında farkettim. düzeltiyorum.

Reply:
yerelde testler başarıyla geçtiği için doğrudan develop dalına gönderdim.
https://github.com/omu/nokul/commit/fb9164633be491b9a630d95c1aac5de1a7316c5b